Makefile

```diff
@@ -251,6 +266,18 @@ hack: hack-push-images hack-build-cli
 		--set worker.image.repository=$(DOCKER_IMAGE_PREFIX)worker \
 		--set worker.image.tag=$(IMMUTABLE_DOCKER_TAG) \
 		--set worker.image.pullPolicy=Always \
+		--set gitInitializer.image.repository=$(DOCKER_IMAGE_PREFIX)git-initializer \
+		--set gitInitializer.image.tag=$(IMMUTABLE_DOCKER_TAG) \
+		--set gitInitializer.image.pullPolicy=Always \
 		--set logger.linux.image.repository=$(DOCKER_IMAGE_PREFIX)logger-linux \
 		--set logger.linux.image.tag=$(IMMUTABLE_DOCKER_TAG) \
 		--set logger.linux.image.pullPolicy=Always
+
+.PHONY: hack-expose-apiserver
+hack-expose-apiserver:
+	@kubectl --namespace brigade port-forward service/brigade-apiserver 7000:443 &>/dev/null & \
+		echo $$! > /tmp/brigade-apiserver.PID
+
+.PHONY: hack-unexpose-apiserver
+hack-unexpose-apiserver:
+	@kill -TERM $$(cat /tmp/brigade-apiserver.PID)
```

Comment:
This is interesting. I've been relying on a tweak I made to my personal `new-kind-cluster` script to handle this, but this is much nicer because it gives this ability to everyone who hacks on this and isn't limited to working with kind.

I like it.